docs/en/wallet-attestation.rst

```diff
@@ -59,6 +59,12 @@ Wallet Instance Initialization and Registration
 
 **Step 1:**: The User starts the Wallet Instance mobile app for the first time.
 
+.. note::
+
+  The WP MUST verify the WI by using the app store vendors API, for Android is Play Integrity API and for iOS is DeviceCheck, these services are defined in this specification as **Device Integrity Service (DIS)**.
+  The EUDIW Application MUST also implement the integrity services from the vendor's SDK, this service has already beed defined as **Device Integrity Service (DIS)** in this specification. The DIS helps by detecting potentially risky and fraudulent interactions, such as from tampered app versions and untrustworthy environments.
+The verification process to establish the trustworthiness of a WI for the WP begins with the initial app launch during which a GUID is generated, serving as the WI's identifier. During the *Initialization and Registration* process, the WI transmits this GUID to the WP, which in turn generates a key pair and signs the GUID with the private key. This signed GUID is then retained by the WI. Subsequently, when the WI requests WIA, it includes the signed GUID. To verify the request's reliability, the WP utilizes the public key generated before to authenticate the GUID.
```

Comment:
where this GUID is defined?

Reply:
i'll change it to UUDI.

It is not defined in the specification.

A UUID, generated by the WSCD, serves as a unique identifier for the application and helps prevent attacks or data manipulation. In this context, the WP can use the UUID to establish trust with a WI.

Reply:
I think you are referring to: `Cryptographic Hardware Key Tag` 
https://github.com/italia/eudi-wallet-it-docs/blob/18a09ac4ad54bc7110259f249526acc6286c648b/docs/en/defined-terms.rst#defined-terms

Reply:
No, the Cryptographic Hardware Key Tag is linked to the generated Cryptographic Hardware Keys and is used to access the private key stored in the hardware.

What I'm referring to is a unique identifier generated by the OS when the WI is launched for the first time, and it exclusively identifies the WI.

Reply:
In our implementation the hardware key tag is also the unique identifier of the Wallet Instance.

> As mentioned in the *Wallet Instance initialization and registration* section above, a Wallet Instance is bound to a Wallet Hardware Key and it's uniquely identified by it.

Reply:
So in this case, i can replace UUID with cryptographic hardware key tag.
What do you think ?